src/components/mainpage/chatscreen/components.mainpage.chatscreen.chatscreen.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from "react";
+import "./components.mainpage.chatscreen.chatscreen.css";
+import Header from "./components.mainpage.chatscreen.header";
+import ChatRoom from "./components.mainpage.chatscreen.chatroom";
+import Form from "./components.mainpage.chatscreen.form";
+
+function ChatScreen() {
+  const [user, setUser] = useState(0);
+  const [UserChat, changeChat] = useState([]);
```

Comment:
```suggestion
  const [userChat, setUserChat] = useState([]);
```
일반적으로 `setUserChat`과 같은 형식을 많이 사용하는 것 같아요